pkg/network/network.go

```diff
@@ -194,41 +193,13 @@ const (
 	HTTPRedirected HTTPProtocol = "redirected"
 )
 
-func validateAndNormalizeOutboundIPRanges(s string) (string, error) {
-	s = strings.TrimSpace(s)
-
-	// * is a valid value
-	if s == "*" {
-		return s, nil
-	}
-
-	cidrs := strings.Split(s, ",")
-	var normalized []string
-	for _, cidr := range cidrs {
-		cidr = strings.TrimSpace(cidr)
-		if len(cidr) == 0 {
-			continue
-		}
-		if _, _, err := net.ParseCIDR(cidr); err != nil {
-			return "", err
-		}
-
-		normalized = append(normalized, cidr)
-	}
-
-	return strings.Join(normalized, ","), nil
-}
-
 // NewConfigFromConfigMap creates a Config from the supplied ConfigMap
 func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 	nc := &Config{}
-	if ipr, ok := configMap.Data[IstioOutboundIPRangesKey]; !ok {
-		// It is OK for this to be absent, we will elide the annotation.
-		nc.IstioOutboundIPRanges = "*"
-	} else if normalizedIpr, err := validateAndNormalizeOutboundIPRanges(ipr); err != nil {
-		return nil, err
-	} else {
-		nc.IstioOutboundIPRanges = normalizedIpr
+	if _, ok := configMap.Data[IstioOutboundIPRangesKey]; ok {
+		// Until the next version is released, the validation check is enabled to notify users who configure some value.
+		logger := logging.FromContext(context.Background()).Named(configMap.Name)
+		logger.Warnf("%q is obsolete as outbound network access is enabled by default now. Remove it from config-network", IstioOutboundIPRangesKey)
```

Comment:
nit: Same here. Deprecated instead of obsolete.

Reply:
Thank you! Done.